src/constants.ts

```diff
@@ -1,4 +1,4 @@
-import type { Facets } from "./url_utils"
+import type { Facets } from "./facet_display/types"
```

Comment:
We should perhaps have separate files for constants and types. Is `DEPARMENTS` the same across search backends? It seems quite specialized and better pulled from the API.